cmd/miniooni/main.go

```diff
@@ -210,29 +204,36 @@ func main() {
 	}
 
 	if !globalOptions.noBouncer {
+		log.Info("Looking up OONI backends")
```

Comment:
This log line is rather mysterious. How can we make it more obvious what we are doing?

Reply:
It doesn't give a good data. We can change the word `backends` to something else. :D

Reply:
Yeah, I agree. What can this something else be?

Reply:
I have no idea